Libplanet.Explorer/GraphTypes/LegacyBencodexValueType.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using Bencodex;
+using GraphQL.Language.AST;
+using GraphQL.Types;
+using Libplanet.Common;
+
+namespace Libplanet.Explorer.GraphTypes
+{
+    public class LegacyBencodexValueType : StringGraphType
```

Comment:
Does we haven't more good naming...? I guess if you have plan about more refactoring, maybe you need LegacyBencodexValueV2Type later. 🙄

Reply:
Well, it's just more work then. To be more sensible and be managable, we need an entirely new set of "V2" scheme which doesn't share the root. Then we'd have to create an entirely new copies of query related codes, management of V1 and V2 graph types, etc. 🙄